Pedro.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjerciciooUno
+{
+    internal class Pedro:Persona
+
+
+    {
+        // se crea metodo para pedro camianr
+        public void caminar ()
+        {
+            Console.WriteLine("pedro esta caminando");
+        }
+
+        //se crea metodo para pedro tocar piano
+        public void tocarPiano()
```

Comment:
Tocar piano following the exercise must recieve a parameter of type String and use it to print the message

Reply:
has been corrected